pkg/cloud/services/elb/loadbalancer.go

```diff
@@ -365,6 +861,20 @@ func ELBName(s scope.ELBScope) (string, error) {
 	return name, nil
 }
 
+// LBName returns the user-defined API Server ELB name, or a generated default if the user has not defined the ELB
+// name.
+// TODO: This might the place where to find out the api server and pass it to the bootstrap script?
```

Comment:
Does this need resolving or could it be answered later?

Reply:
Ah, no we don't need this actually. I shall remove this todo. I made it work without hosts rewrite. :)